tests/privategroups_test.go

```diff
@@ -181,7 +181,7 @@ func XTestGroupsJSCreate(t *testing.T) {
 			})
 		}, 2000)
 	}))
-`, alice.Ref(), helloGroup.Ref()), ``)
+`, alice.Sigil(), helloGroup.Ref()), ``)
```

Comment:
there's a missed `.Ref()` here, not sure if that's intended or simply something that was not caught by the search & replace